mobile/Screen/EventDetailScreen.js

```diff
@@ -23,27 +25,70 @@ const styles = StyleSheet.create({
     backgroundColor: '#D9D9D9',
     margin: 40,
   },
-  border: {
-    borderBottomColor: '#8A8A8A',
-    borderBottomWidth: '1.5',
-    marginVertical: 30,
-    marginBottom: 30,
-    marginRight: 20,
-    marginLeft: 20,
-  },
+  // border: {
+  //   borderBottomColor: '#8A8A8A',
+  //   borderBottomWidth: '1.5',
```

Comment:
It looks like the line causing an error was `borderBottomWidth: '1.5',`, so removing that from Angela's code should work. But it's generally better practice to use flex (as you did) than hardcoding margin values, so I think you can remove this commented-out`border` section.